rt_factory/rt_factory.py

```diff
@@ -153,3 +158,53 @@ def add_private_key(self, filename, phrase):
         path = 'gpg/key/private'
         self._put_file(path, filename, headers={'X-GPG-PASSPHRASE': phrase})
 
+    def get_link_to_last_modified(self, repo, path):
+        """ Searches for artifacts with the latest modification date.
+
+        Args:
+            repo (str): The repository name.
+            path (str): The full path to the file to be searched for.
+
+        Returns:
+            str: Download url to the artifact.
+        """
+        the_link = self._get('storage/'+repo+'/'+path+'/?lastModified')['uri']
+        return self._get_from_url(the_link)['downloadUri']
+
+    def get_link_to_last_version(self, repo, path):
+        """ Searches for artifacts with the latest value in the "version" property. Only artifacts
+        with a "version" property expressly defined in lower case will be taken into account. 
```

Comment:
What do you mean by this? Will it give issue with version numbers like "V1.0.0" or "1.4.8-BETA"?